inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_b_fs_zyx_fsv16_imad.cl

```diff
@@ -146,11 +219,11 @@ KERNEL(convolution_gpu_b_fs_zyx_fsv16_imad)(
                                         if (v + in_f_offset < FSV) {
                                             input_int8_arr[v] = conv_input[input_idx + tmp * FSV + v];
                                         } else {
-                                            input_int8_arr[v] = conv_input[input_idx + tmp * FSV + v + 
-                                                                           ((INPUT0_SIZE_X + 2*PADDING_SIZE_X) * 
-                                                                            (INPUT0_SIZE_Y + 2*PADDING_SIZE_Y) * 
-                                                                            (INPUT0_SIZE_Z + 2*PADDING_SIZE_Z) - 1) * 
-                                                                           FSV];
+                                            const uint addr = input_idx + tmp * FSV + v +
+                                                        ((INPUT0_SIZE_X + INPUT0_PAD_BEFORE_SIZE_X + INPUT0_PAD_AFTER_SIZE_X) *
+                                                         (INPUT0_SIZE_Y + INPUT0_PAD_BEFORE_SIZE_Y + INPUT0_PAD_AFTER_SIZE_Y) *
+                                                         (INPUT0_SIZE_Z + INPUT0_PAD_BEFORE_SIZE_Z + INPUT0_PAD_AFTER_SIZE_Z) - 1) * FSV;
```

Comment:
Could you extract accuracy fix into separate PR?

Reply:
Done in #2876